docs/connect.html

```diff
@@ -369,6 +369,114 @@ <h4><a id="connect_errorreporting" href="#connect_errorreporting">Error Reportin
 # Tolerate all errors.
 errors.tolerance=all</pre>
 
+    <h4><a id="connect_exactlyonce" href="#connect_exactlyonce">Exactly-once support</a></h4>
+
+    <p>Kafka Connect is capable of providing exactly-once delivery guarantees for sink connectors (as of version 0.11.0) and source connectors (as of version 3.3.0). Please note that <b>support for exactly-once delivery is highly dependent on the type of connector you run.</b> Even if you set all the correct worker properties in the configuration for each node in a cluster, if a connector is not designed to, or cannot take advantage of the capabilities of the Kafka Connect framework, exactly-once may not be possible.</p>
```

Comment:
> exactly-once delivery guarantees

We should not use this phrase... it's loaded...

"delivery" is a coined term from a research paper that proves that it's not possible to send a message exactly once (ie, network layer) -- EOS in Kafka is not about "not retrying (or only retrying if we magically know that the message was not delivered" but to "mask" retries and provide the same "semantic result in case of failure and retries".

Reply:
Seems "delivery" is used on multiple other places further below, too.

Reply:
Thanks Matthias, I didn't realize this term could be so controversial. I've filed https://github.com/apache/kafka/pull/13106 as a follow-up to fix the wording used here and elsewhere in Connect.